libs/zpool/build.zig

```diff
@@ -1,10 +1,8 @@
 const std = @import("std");
 
-const main = .{ .zig = thisDir() ++ "/src/main.zig" };
```

Comment:
I thought perhaps things weren't working because I tried to be clever here, but changing this does not seem to have any effect.